plugins/package-managers/node/src/main/kotlin/NodePackageManager.kt

```diff
@@ -33,12 +33,15 @@ import org.ossreviewtoolkit.model.Project
 import org.ossreviewtoolkit.model.ProjectAnalyzerResult
 import org.ossreviewtoolkit.model.config.AnalyzerConfiguration
 import org.ossreviewtoolkit.model.utils.DependencyGraphBuilder
+import org.ossreviewtoolkit.utils.common.DirectoryStash
 import org.ossreviewtoolkit.utils.common.realFile
 
 const val NODE_MODULES_DIRNAME = "node_modules"
 const val NPM_RUNTIME_CONFIGURATION_FILENAME = ".npmrc"
 
 abstract class NodePackageManager(val managerType: NodePackageManagerType) : PackageManager(managerType.projectType) {
+    protected lateinit var dirStash: DirectoryStash
```

Comment:
When this class was introduced, there was this agreement to not move back into the direction of using this parent class for sharing a lot of code, but only share the parsing of projects. 

But now, this seems to start growing. I fear this moves towards inheritance related complexity, and would revert the gains from separating things. What do you think?

Reply:
ok, I see we're on same page. Thinking a bit about it, I was more concerned about potential future additions going into this direction, than about this PR. So, let's do this.